x-pack/plugins/security_solution/public/management/components/endpoint_responder/components/header_endpoint_info.tsx

```diff
@@ -88,7 +88,11 @@ export const HeaderEndpointInfo = memo<HeaderEndpointInfoProps>(({ endpointId })
                 id="xpack.securitySolution.responder.header.lastSeen"
                 defaultMessage="Last seen {date}"
                 values={{
-                  date: <FormattedRelative value={endpointDetails.metadata['@timestamp']} />,
+                  date: endpointDetails.last_checkin ? (
+                    <FormattedRelative value={endpointDetails.last_checkin} />
+                  ) : (
+                    '-'
```

Comment:
So `FormattedRelative` is a different component that expects a string that can be parsed into a date. So I'll still have to use the check here. I'd use the empty value instead of the hardcoded `-` though